### PR TITLE
Update NEWS to Markdown format

### PR DESCRIPTION
**Pull request by dan-knight**
## Description

These changes update `NEWS` to the `NEWS.md` format for consistency with other lab packages. Also standardizes headings to use [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) standard terminology (Changed, Added, etc.).

## Checklist

-   [x] This PR does NOT contain [PHI](https://ohrpp.research.ucla.edu/hipaa/) or germline genetic data. A repo may need to be deleted if such data is uploaded. Disclosing PHI is a [major problem](https://healthitsecurity.com/news/ucla-health-reaches-7.5m-settlement-over-2015-breach-of-4.5m).

-   [x] This PR does NOT contain molecular files, compressed files, output files such as images (*e.g.* `.png`, .`jpeg`), `.pdf`, `.RData`, `.xlsx`, `.doc`, `.ppt`, or other non-plain-text files. To automatically exclude such files using a [.gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) file, see [here](https://github.com/uclahs-cds/template-base/blob/main/.gitignore) for example.

-   [x] I have read the [code review guidelines](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3187646/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List).

-   [x] I have set up or verified the `main` branch protection rule following the [github standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3190380/GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.

-   [x] The name of the branch is meaningful and well formatted following the [standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List), using [AD_username (or 5 letters of AD if AD is too long)]-[brief_description_of_branch].

-   [x] I have added the major changes included in this pull request to the `NEWS` under the next release version or unreleased, and updated the date. I have also updated the version number in `DESCRIPTION` according to [semantic versioning](https://semver.org/) rules.

-   [ ] Both `R CMD build` and `R CMD check` run successfully.

### Case 1

``` r
# input code
```

### Case 2

``` r
# input code
```

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,4111 @@
+# BoutrosLab.plotting.general 7.1.2 (2024-10-02)
+
+## Changed
```
Comment (aholmes) [inline NEWS.md:3]: **suggestion (blocking):** Types of changes are at the third level `###`

**Review by aholmes — CHANGES_REQUESTED** [review]
**question:** Is this change acceptable with CRAN?

File: NEWS.md
[the remark this answers is not in the document]
Comment (dan-knight) [reply NEWS.md:1]: Good catch.

**Review by aholmes — APPROVED** [review]
Ignore my comments, you clarified that the format differences are so R CMD Check passes.

**Pull request merged**